app/qml/components/FieldRow.qml

```diff
@@ -26,11 +26,11 @@ Item {
     InputTextField {
       id: textField
       color: fieldDelegate.color
-      text: AttributeName
       Layout.fillHeight: true
       Layout.fillWidth: true
       Layout.preferredWidth: row.itemSize
 
+      Component.onCompleted: text = AttributeName
       onDisplayTextChanged: AttributeName = displayText
```

Comment:
http://imaginativethinking.ca/bi-directional-data-binding-qt-quick/

instead of onDisplayTextChanged signal it is better to do binding  AttributeName = textField.text

Reply:
Changed for simple textChanged signal handling